handlers/image.js

```diff
@@ -0,0 +1,34 @@
+module.exports = browser => async (req, res) => {
+  let page = await browser.newPage()
+  let url = req.body.url
+
+  if (!url) res.status(400).send()
+
+  try {
+    await page.goto(url)
+    await page.addScriptTag({ path: `./lib/customDomToImage.js` })
+
+    const targetElement = await page.$('#export-container')
+
+    let dataUrl = await page.evaluate(target => {
```

Comment:
Should we pull this out into a lib function so we don't always have to update it in two places?